app/controllers/price_groups_controller.rb

```diff
@@ -28,7 +28,7 @@ def show
 
   # GET /facilities/:facility_id/price_groups/:id/users
   def users
-    unless @price_group_ability.can?(:read, UserPriceGroupMember)
+    unless SettingsHelper.feature_on?(:user_based_price_groups) && @price_group_ability.can?(:read, UserPriceGroupMember)
```

Comment:
No; really the [specs for this controller](https://github.com/tablexi/nucore-open/blob/cc49546d164be98308dade4c441ed38b85609f64/spec/controllers/price_groups_controller_spec.rb) could use some attention.